3rdParty/fuerte/src/Basics/Format.h

```diff
@@ -0,0 +1,115 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2018 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Simon Grätzer
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef ARANGO_CXX_DRIVER_FORMAT_H
+#define ARANGO_CXX_DRIVER_FORMAT_H 1
+
+#include "Endian.h"
+#include "operating-system.h"
+
+// enable unalgined little-endian data access
+#undef TRI_USE_FAST_UNALIGNED_DATA_ACCESS
+#ifdef TRI_UNALIGNED_ACCESS
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#define TRI_USE_FAST_UNALIGNED_DATA_ACCESS
+#endif
+#endif
+
+namespace arangodb {
+namespace basics {
```

Comment:
mmh, uses the same namespace as ArangoDB...
should be moved into Fuerte's own namespace to avoid conflicts.